[libcap] Failed to download from https://git.kernel.org/pub/scm/libs/libcap

Package: libcap:x64-linux@2.69#3

**Host Environment**

- Host: x64-linux
- Compiler: GNU 7.5.0
-    vcpkg-tool version: 2023-12-12-1c9ec1978a6b0c2b39c9e9554a96e3e275f7556e
    vcpkg-scripts version: 6c296b954 2024-03-29 (7 hours ago)

**To Reproduce**

`vcpkg install libcap`
**Failure logs**

```
-- Downloading https://git.kernel.org/pub/scm/libs/libcap/libcap.git/snapshot/libcap-3c7dda330bd9a154bb5b878d31fd591e4951fe17.tar.gz -> libpcap-2.69.tar.gz...
[DEBUG] To include the environment variables in debug output, pass --debug-env
[DEBUG] Trying to load bundleconfig from /home/a/vcpkg/vcpkg-bundle.json
[DEBUG] Failed to open: /home/a/vcpkg/vcpkg-bundle.json
[DEBUG] Bundle config: readonly=false, usegitregistry=false, embeddedsha=nullopt, deployment=Git, vsversion=nullopt
[DEBUG] Metrics enabled.
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] Feature flag 'dependencygraph' unset
[DEBUG] 1000: execute_process(curl --fail -L https://git.kernel.org/pub/scm/libs/libcap/libcap.git/snapshot/libcap-3c7dda330bd9a154bb5b878d31fd591e4951fe17.tar.gz --create-dirs --output /home/a/vcpkg/downloads/libpcap-2.69.tar.gz.35688.part)
[DEBUG] 1000: cmd_execute_and_stream_data() returned 22 after   784363 us
error: Failed to download from mirror set
error: https://git.kernel.org/pub/scm/libs/libcap/libcap.git/snapshot/libcap-3c7dda330bd9a154bb5b878d31fd591e4951fe17.tar.gz: curl failed to download with exit code 22
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

curl: (22) The requested URL returned error: 404

[DEBUG] /mnt/vss/_work/1/s/src/vcpkg/base/downloads.cpp(1048):
[DEBUG] Time in subprocesses: 784363us
[DEBUG] Time in parsing JSON: 15us
[DEBUG] Time in JSON reader: 0us
[DEBUG] Time in filesystem: 528us
[DEBUG] Time in loading ports: 0us
[DEBUG] Exiting after 787 ms (785547us)

CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:32 (message):

      Failed to download file with error: 1
      If you are using a proxy, please check your proxy setting. Possible causes are:

      1. You are actually using an HTTP proxy, but setting HTTPS_PROXY variable
         to `https://address:port`. This is not correct, because `https://` prefix
         claims the proxy is an HTTPS proxy, while your proxy (v2ray, shadowsocksr
         , etc..) is an HTTP proxy. Try setting `http://address:port` to both
         HTTP_PROXY and HTTPS_PROXY instead.

      2. If you are using Windows, vcpkg will automatically use your Windows IE Proxy Settings
         set by your proxy software. See https://github.com/microsoft/vcpkg-tool/pull/77
         The value set by your proxy might be wrong, or have same `https://` prefix issue.

      3. Your proxy's remote server is out of service.

      If you've tried directly download the link, and believe this is not a temporary
      download server failure, please submit an issue at https://github.com/Microsoft/vcpkg/issues
      to report this upstream download server failure.


Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:270 (z_vcpkg_download_distfile_show_proxy_and_fail)
  ports/libcap/portfile.cmake:1 (vcpkg_download_distfile)
  scripts/ports.cmake:175 (include)



```

--- a/ports/libcap/portfile.cmake
+++ b/ports/libcap/portfile.cmake
@@ -1,7 +1,7 @@
 vcpkg_download_distfile(ARCHIVE
-    URLS https://git.kernel.org/pub/scm/libs/libcap/libcap.git/snapshot/libcap-3c7dda330bd9a154bb5b878d31fd591e4951fe17.tar.gz
-    FILENAME libpcap-${VERSION}.tar.gz
-    SHA512 0732c9f07be38c2bb06409f1f31d7ed5ad31b38ae380650b334074856f89b885deabb9603e21a989e81b530050c068bbbf60157adbf3ca3893e4bca7d61f63d2
+    URLS "https://git.kernel.org/pub/scm/libs/libcap/libcap.git/snapshot/libcap-${VERSION}.tar.gz"
+    FILENAME "libcap-${VERSION}.tar.gz"
+    SHA512 b243ae403d45af2ff1204931b9e24c3b7f3e0c444f1ff2f3ed524c212b61a7ff1bc3c98df1855b2f0d300ebabf604b95440cdaddd666914ad60575e2e2f29fe8
 )
 
 vcpkg_extract_source_archive(SOURCE_PATH

--- a/ports/libcap/vcpkg.json
+++ b/ports/libcap/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "libcap",
   "version": "2.69",
-  "port-version": 3,
+  "port-version": 4,
   "description": "A library for getting and setting POSIX.1e (formerly POSIX 6) draft 15 capabilities.",
   "homepage": "https://git.kernel.org/pub/scm/libs/libcap/libcap.git",
   "license": "BSD-3-Clause OR GPL-2.0-only",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4198,7 +4198,7 @@
     },
     "libcap": {
       "baseline": "2.69",
-      "port-version": 3
+      "port-version": 4
     },
     "libcbor": {
       "baseline": "0.11.0",

--- a/versions/l-/libcap.json
+++ b/versions/l-/libcap.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "f81a07cf694095f60c759c9a18adc57e94296a5f",
+      "version": "2.69",
+      "port-version": 4
+    },
+    {
       "git-tree": "23bb9f2743e684bacba57b8cc8a931756f0e7447",
       "version": "2.69",
       "port-version": 3

